component/gitlab-ci.jsonnet

```diff
@@ -96,7 +92,7 @@ local GitLabCI() = {
     environment: {
       name: 'production',
     },
-    variables: cloud_specific_variables[params.provider].apply,
+    [if std.length(std.get(cloud_specific_variables[params.provider], 'apply', {})) > 0 then 'variables']: cloud_specific_variables[params.provider].apply,
```

Comment:
I would probably just make `apply` an empty object for Exoscale then you don't need all this here.